add-on/preferences/prefs.js

```diff
@@ -33,11 +33,7 @@ pref("loop.CSP", "default-src 'self' about: file: chrome:; img-src * data:; font
 pref("loop.fxa_oauth.tokendata", "");
 pref("loop.fxa_oauth.profile", "");
 pref("loop.support_url", "https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/cobrowsing");
-#ifdef LOOP_BETA
 pref("loop.facebook.enabled", true);
```

Comment:
I'd rather keep it simple to disable if we need to. It isn't that much extra code compared to the rest and it shouldn't hurt performance.

I've filed [bug 1252830](https://bugzilla.mozilla.org/show_bug.cgi?id=1252830) on removing it later on.